test/unit/tech/html5.test.js

```diff
@@ -388,9 +388,11 @@ if (Html5.supportsNativeTextTracks()) {
       addEventListener: (type, fn) => events.push([type, fn]),
       removeEventListener: (type, fn) => events.push([type, fn])
     };
-    const el = document.createElement('div');
+    const el = document.createElement('video');
```

Comment:
is this related to test failures?

Reply:
It is. I am not sure about the assert below anymore, but it was causing some weird error in IE11 of all things, could be worth re-investigating the assert again. This change seems harmless though.

Reply:
This may have also been due to the memory issues during testing, maybe this can all be reverted

Reply:
Ah, this is because we're getting the `src` getter from the media element and applying it to the div which safari doesn't like. So, unless we update setting the src getter code, I guess this piece is fine to leave.